tests/unit_tests/test_power.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def power_process_code():
+    return load_process_code("power")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({'p': 1, 'base': 2}, 2),
+        ({'p': 0, 'base': 1234}, 1),
```

Comment:
test for base 0 could also be added
```
({"p": 2, "base": 0}, 0),
```
Make sure that you have black setup correctly or run (also for other tests) 

```
pipenv run black -l 120 .
```

fftr

Reply:
Fixed